### PR TITLE
Fix spec for offers controller search

### PR DESCRIPTION
**Pull request by rewritten**

### CODE REVIEW

**Comment by rewritten** [comment]
:green_heart: :green_heart: :green_heart:

File: spec/controllers/offers_controller_spec.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@
       login(another_member.user)
 
       get "index", q: offer.title.split(/\s/).first
-      expect(assigns(:offers)).to eq([offer])
+      expect(assigns(:offers).to_a).to eq([offer])
```
Comment (enricostano) [inline spec/controllers/offers_controller_spec.rb]: what returns `assigns(:offers)`? why do we need the `.to_a`?

Comment (rewritten) [inline spec/controllers/offers_controller_spec.rb]: It returns a proxy from ElasticSearch to a database recordset. It delegates almost everything to the underlying recordset, like `:[]`, `:to_a`, `:each` etc. But it is **not** an array itself.

Comment (rewritten) [inline spec/controllers/offers_controller_spec.rb]: I'll add a comment.

**Comment by enricostano** [comment]
:+1: :clap:

**Pull request merged**